src/configure/index.js

```diff
@@ -13,7 +13,29 @@ export default function configure (config = {}) {
   return [
     path,
     loadUserConfig,
-    webpackConfig,
-    karmaConfig
+    webpack,
+    karma
   ].reduce((config, plugin) => plugin(config), config)
 }
+
+export function configureWebpack (userWebpackConfig = {}) {
+  const { sagui: config, ...webpackConfig } = userWebpackConfig
+
+  return [
+    path,
+    webpack
+  ].reduce((config, plugin) => plugin(config), { ...config, webpackConfig }).webpackConfig
+}
+
+export function configureKarma (userWebpackConfig, userKarmaConfig = {}) {
```

Comment:
What do you think about defaulting `userWebpackConfig` to `{}`  in case the input is not an object? It is user-provided data after all.